artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AmqpCoreConverter.java

```diff
@@ -220,7 +224,6 @@ public static ICoreMessage toCore(AMQPMessage message, CoreMessageObjectPools co
          }
       }
 
-      result.getInnerMessage().setReplyTo(message.getReplyTo());
```

Comment:
I think I added this due to a bug. Can you check history by selection, I would have added test case and check it’s passing.

Reply:
The previous set was definitely buggy. that's why you added this... but this was always setting it to Queue.

I'm doing a more through check now.

Reply:
I will run a full testsuite on this.

Reply:
sounds good

Reply:
everything passing ok, I will resolve this!